modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Details.tsx

```diff
@@ -0,0 +1,165 @@
+	const [labelValidationError, setLabelValidationError] = useState(false);
+
+	const fdsViewDescriptionRef = useRef<HTMLInputElement>(null);
+	const fdsViewLabelRef = useRef<HTMLInputElement>(null);
+
+	const updateFDSView = async () => {
+		const body = {
+			description: fdsViewDescriptionRef.current?.value,
+			label: fdsViewLabelRef.current?.value,
+		};
+
+		const response = await fetch(`${fdsViewsAPIURL}/${fdsView.id}`, {
+			body: JSON.stringify(body),
+			headers: {
+				'Accept': 'application/json',
+				'Content-Type': 'application/json',
+			},
+			method: 'PATCH',
+		});
+
+		const responseJSON = await response.json();
+
+		if (responseJSON?.id) {
+			openToast({
+				message: Liferay.Language.get(
+					'your-request-completed-successfully'
+				),
+				type: 'success',
+			});
+
+			const controlMenuHeaderTitles = document.getElementsByClassName(
+				'control-menu-level-1-heading'
+			);
+
+			if (controlMenuHeaderTitles.length === 1) {
+				controlMenuHeaderTitles[0].innerHTML = Liferay.Util.escapeHTML(
+					fdsViewLabelRef.current?.value ?? ''
+				);
+			}
+		}
+		else {
+			openToast({
+				message: Liferay.Language.get(
+					'your-request-failed-to-complete'
+				),
+				type: 'danger',
+			});
+		}
+	};
+
+	return (
+		<ClayLayout.Sheet className="mt-3" size="lg">
+			<ClayLayout.SheetHeader>
+				<h2 className="sheet-title">
+					{Liferay.Language.get('details')}
+				</h2>
+			</ClayLayout.SheetHeader>
+
+			<ClayLayout.SheetSection>
+				<ClayForm.Group
+					className={classNames({
+						'has-error': labelValidationError,
+					})}
+				>
+					<label htmlFor={`${namespace}fdsViewLabelInput`}>
+						{Liferay.Language.get('name')}
+
+						<RequiredMark />
+					</label>
+
+					<ClayInput
+						defaultValue={fdsView.label}
+						id={`${namespace}fdsViewLabelInput`}
+						onBlur={() => {
+							setLabelValidationError(
+								!fdsViewLabelRef.current?.value
+							);
+						}}
+						ref={fdsViewLabelRef}
+						type="text"
+					/>
+
+					{labelValidationError && (
+						<ClayForm.FeedbackGroup>
+							<ClayForm.FeedbackItem>
+								<ClayForm.FeedbackIndicator symbol="exclamation-full" />
+
+								{Liferay.Language.get('this-field-is-required')}
+							</ClayForm.FeedbackItem>
+						</ClayForm.FeedbackGroup>
+					)}
+				</ClayForm.Group>
+
+				<ClayForm.Group>
+					<label htmlFor={`${namespace}fdsViewDesctiptionInput`}>
+						{Liferay.Language.get('description')}
+					</label>
+
+					<ClayInput
+						defaultValue={fdsView.description}
+						id={`${namespace}fdsViewDesctiptionInput`}
+						ref={fdsViewDescriptionRef}
+						type="text"
+					/>
+				</ClayForm.Group>
+			</ClayLayout.SheetSection>
+
+			<ClayLayout.SheetFooter>
+				<ClayButton.Group spaced>
+					<ClayButton onClick={updateFDSView}>
+						{Liferay.Language.get('save')}
+					</ClayButton>
+
+					<ClayButton
+						displayType="secondary"
+						onClick={() => {
+							navigate(fdsViewsURL);
+						}}
```

Comment:
Obsolete braces, might be wrong, formatter might be adding them due to line width